examples/jsm/loaders/obj2/worker/main/WorkerExecutionSupport.js

```diff
@@ -0,0 +1,444 @@
+/**
+ * @author Kai Salmen / https://kaisalmen.de
+ * Development repository: https://github.com/kaisalmen/WWOBJLoader
+ */
+
+/**
+ * These instructions are used by {WorkerExecutionSupport} to build code for the web worker or to assign code
+ *
+ * @param {boolean} supportsStandardWorker
+ * @param {boolean} supportsJsmWorker
+ * @constructor
+ */
+const CodeBuilderInstructions = function ( supportsStandardWorker, supportsJsmWorker, preferJsmWorker ) {
+	this.supportsStandardWorker = supportsStandardWorker;
+	this.supportsJsmWorker = supportsJsmWorker;
+	this.preferJsmWorker = preferJsmWorker;
+	this.startCode = '';
+	this.codeFragments = [];
+	this.importStatements = [];
+
+	this.jsmWorkerFile;
```

Comment:
Just declaring a property of the newly constructed object without initializing it is no good approach. As mentioned by lgtm, this is unnecessary since the property will be created upon first assignment. 

Assigning a default value will also make the type of this property more clear.

Reply:
Yeah, overlooked it. Thanks. I will fix this